remove_test.go

```diff
@@ -0,0 +1,100 @@
+//Copyright 2013 GoGraphviz Authors
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+package gographviz
+
+import (
+	"fmt"
+)
+
+func ExampleGraph_RemoveNode() {
```

Comment:
Maybe we could add a few more tests?

Reply:
You're right, will improve this with more tests.

Reply:
Oops, I feel like a noob. I was under the impression that I added a test. But it was an example, with output verification. Changed it to be a test instead :) Hope it's good now.

Reply:
I had no problem with the example, but I guess it is better as test.
And don't get me wrong this test covers a lot of cases, it is really cool.

I was wondering if you could add another test, that leaves some of the original nodes in the graph that had a connecting edge with the node that was removed.

Reply:
Added more checkpoints and tests. Rebased the PR. Happy? :D

Reply:
Yes very :D

Reply:
Thank you so much